packages/react/src/components/DateTimePicker/_date-time-picker.scss

```diff
@@ -121,70 +125,102 @@
       &.#{$iot-prefix}--date-time-picker__menu-expanded {
         display: block;
       }
+    }
+  }
 
-      .#{$iot-prefix}--date-time-picker__menu-scroll {
```

Comment:
This diff looks much worse than it actually is. I split the `.#{$iot-prefix}--date-time-picker__box` into 2 separate classes `.#{$iot-prefix}--date-time-picker__box--icon-only` and `.#{$iot-prefix}--date-time-picker__box--full` for the unique differences. I then took the common nested classes and moved them outside of the scope of the boxes to avoid duplication